sdk/cognitiveservices/azopenai/examples_client_test.go

```diff
@@ -53,8 +55,10 @@ func ExampleNewClient() {
 func ExampleNewClientWithKeyCredential() {
 	// NOTE: this constructor creates a client that connects to an Azure OpenAI endpoint.
 	// To connect to the public OpenAI endpoint, use azopenai.NewClientForOpenAI
-	keyCredential := azopenai.KeyCredential{
-		APIKey: "Azure OpenAI apikey",
+	keyCredential, err := azopenai.NewKeyCredential("Azure OpenAI apikey", nil)
```

Comment:
All your examples use qa different string parameter: "open-ai-apikey" or "Azure OpenAI apikey". Please be consistent and if the customer must replace this value, then put it in angle brackets or something, like:<OpenAI-APIKey> or <Azure-OpenAI-APIKey>

Reply:
Fixed.